tests/src/Core/Path/PathautoGeneratorTraitTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace drunomics\ServiceUtils\Tests\Core\Path;
```

Comment:
the namespace and directory should follow Drupal\pathauto\PathautoGeneratorInterface -  thus drunomics\ServiceUtils\Tests\pathauto\PathautoGeneratorInterface

you might have to enable the module in the setup method such that the modules namespace is registered and code autloading works.

Reply:
The problem is that pathauto is a contrib module and to be able to use it in the test I need to require this module in composer.json that might be not a good idea. I don't sure if we have to use services from contrib modules in service-utils

Reply:
oh I see. well, we could simply add it to require-dev - that stuff is going to be installed during testing anyway.